segmentstore/server/src/test/java/io/pravega/segmentstore/server/writer/WriterConfigTest.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.segmentstore.server.writer;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Writer Config Test
+ */
+public class WriterConfigTest {
+
+    @Test
+    public void testMaxRolloverSize() {
```

Comment:
Thanks, that's great. One comment: given that we are not testing any default value for this config class, can we change a bit the test to be something like `testDefaultValues()` and include in the test all the other values apart from max rollover size?

Reply:
Sure Raul, I will do the changes.